bridged-nft-token/src/nft_core.rs

```diff
@@ -0,0 +1,289 @@
+use crate::*;
```

Comment:
Unless there is a significant difference, let's use macro to derive nft core implementation from [near contract standards](https://github.com/near/near-sdk-rs/blob/master/near-contract-standards/src/non_fungible_token/macros.rs#L4)

Reply:
it was removed and replaced by the one contained inside the standard